.github/workflows/verify-prs.yml

```diff
@@ -0,0 +1,31 @@
+name: Verify PRs
+
+on:
+  pull_request:
+    branches: ['master']
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Check out the code
+      - uses: actions/checkout@v2
+
+        # Enable caching of Maven dependencies to speed up job execution.  See https://github.com/actions/cache
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+        # Set up Java 1.8 with Maven
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+        # Execute the Maven verify command to compile, package, test, verify
+      - name: Build with Maven
+        run: mvn -B verify --file pom.xml
```

Comment:
@solemabrothers Why not `mvn clean install`

Reply:
Done